lib/react_client/react_interop.dart

```diff
@@ -113,7 +113,7 @@ class ReactElementStore {
 @JS()
 @anonymous
 class ReactElement {
-  external ReactElementStore get _store;
+  external ReactElementStore get _store; // ignore: unused_element
```

Comment:
If this is private and unused do we even need to keep it? If so we should probably add a code comment explaining why it's still around.